triplea_maps.yaml

```diff
@@ -327,6 +327,29 @@
     <br>Feedback highly welcome, if played.
     <br>Leave feedback at the forums: http://triplea.sourceforge.net/mywiki/Community
     <br>or at the Depot: http://sites.google.com/site/tripleaerniebommel/
++  url: https://github.com/KaiMAD/big_world_1939/blob/master/big_world_1939.zip+
```

Comment:
yeah, sorry about the trouble, it's supposed to be an easy process. thanks @KaiMAD